app/src/org/commcare/google/services/ads/AdMobManager.java

```diff
@@ -0,0 +1,118 @@
+package org.commcare.google.services.ads;
+
+import android.content.Context;
+import android.view.View;
+import android.widget.FrameLayout;
+
+import com.google.android.gms.ads.AdRequest;
+import com.google.android.gms.ads.AdSize;
+import com.google.android.gms.ads.AdView;
+import com.google.android.gms.ads.MobileAds;
+
+import org.commcare.CommCareApplication;
+import org.commcare.dalvik.BuildConfig;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * @author Aliza Stone
+ */
+public class AdMobManager {
+
+    private static final String AD_TYPE_BANNER = "banner";
+    private static final String AD_TYPE_INTERSTITIAL = "interstitial";
+
+    private static final Map<String, String> adMobIdsMap = initAdMobIdsMap();
+
+    private static final String CLINICAL_SCALES_PACKAGE_ID =
+            "org.commcare.consumerapps.clinicalscales";
+
+    public static Map<String, String> initAdMobIdsMap() {
+        Map<String, String> map = new HashMap<>();
+        map.put(CLINICAL_SCALES_PACKAGE_ID, "ca-app-pub-8038725004530429~5040587593");
```

Comment:
A few magic strings in here - would be more readable if they were moved into constants.